### ISSUE TITLE
[Enhancement] Version Code

### ISSUE THREAD

**Issue by black-cerberon** [microg/android_packages_apps_GmsCore#1081]
Please use higher version code (2019420020 instead 19420020), because the original Google Play Service from Play Market overwrites it everytime.

**Comment by ArchangeGabriel** [comment]
I don’t think this can be done, because some things actually rely on this number. You should blacklist update of Play Services instead.

**Comment by black-cerberon** [comment]
> You should blacklist update of Play Services instead.

Play Services can't be blacklisted from original Play Market.

**Comment by ArchangeGabriel** [comment]
What about automatic updates then? Can they be disabled?

**Comment by black-cerberon** [comment]
It's possible, but not reliable.

> I don’t think this can be done, because some things actually rely on this number.

What you talking about?

**Comment by chris42** [comment]
I am not sure, if this is all correct. The versions you stated seem to be a different formatting: 2019 instead of 19?
If it is just about the formatting, then that might be picked up by @mar-v-in 

However, even then the error within the regular play store will remain, as microgs version number is pretty much always behind the original google service framework. This has been discussed several times before.

I would advise to use an alternative play store if automatic updates can't be disabled, e.g. like aurora.

**Comment by ArchangeGabriel** [comment]
@chris42 No, you did not get @black-cerberon issue. He wants microG to advertise a version that is always higher than the official one (hence the different formatting).

@black-cerberon Some app check feature availability based on version number for instance. If you advertise a version number that is higher than the one those features were implemented in, but the features actually don’t exist in microG, this could lead to crashes (and it actually does).

**Comment by chris42** [comment]
@ArchangeGabriel Ok, I thought @black-cerberon had seen the 2019 somewhere, e.g. because google changed the formatting.
